src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -194,14 +204,17 @@ public static AbstractBuilding createFromNBT(Colony colony, NBTTagCompound compo
             }
             catch (RuntimeException ex)
             {
-                Log.logger.error(String.format("A Building %s(%s) has thrown an exception during loading, its state cannot be restored. Report this to the mod author",
-                                        compound.getString(TAG_BUILDING_TYPE), oclass.getName()), ex);
+                Log.logger.error(String
+                        .format("A Building %s(%s) has thrown an exception during loading, its state cannot be restored. Report this to the mod author", compound
+                                .getString(TAG_BUILDING_TYPE), oclass.getName()), ex);
                 building = null;
             }
         }
         else
         {
-            Log.logger.warn(String.format("Unknown Building type '%s' or missing constructor of proper format.", compound.getString(TAG_BUILDING_TYPE)));
+            Log.logger
```

Comment:
Agree with Marvin, will need to increase the line length. Also please don't format a whole class, it makes PR's harder to review